.github/workflows/tests.yml

```diff
@@ -19,87 +19,46 @@ jobs:
         PYTHON_VERSION: ["3.11", "3.12"]
     steps:
       - uses: actions/checkout@v4
-      - name: Checkout fulcrumgenomics/bwa-aln-interactive
-        uses: actions/checkout@v4
-        with:
-          repository: fulcrumgenomics/bwa-aln-interactive
-          ref: main
-          path: bwa
-          fetch-depth: 0
 
       - name: Set up Python ${{ matrix.PYTHON_VERSION }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.PYTHON_VERSION }}
 
+      - name: Install poetry
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install poetry==${{env.POETRY_VERSION}}
```

Comment:
**suggestion** Should we use `pipx` to install `poetry` so it gets its own environment?